index.html

```diff
@@ -14,19 +14,23 @@
     </header>
     <main>
       <div>
-        <h2>Hi, l'm Scotty.</h2>
+        <h2>Hi, l am Scotty.</h2>
```

Comment:
```suggestion
        <h2>Hi, I am Scotty.</h2>
```

Alphabet 아이 가 아니라 엘 이 들어갔네요